[0m[[0m[31merror[0m] [0m[0m/home/s4n/Documents/corrientazo-domicilio/src/test/scala/co/com/corrientazoadomicilio/test/testSuite.scala:149:17: value isFailure is not a member of List[scala.util.Try[co.com.corrientazoadomicilio.entidades.Dron]][0m
[0m[[0m[31merror[0m] [0m[0m    assert(res3.isFailure)[0m
[0m[[0m[31merror[0m] [0m[0m                ^[0m
[0m[[0m[31merror[0m] [0m[0mone error found[0m
